.palantir/revapi.yml

```diff
@@ -1253,6 +1253,97 @@ acceptedBreaks:
       old: "method void org.apache.iceberg.data.parquet.BaseParquetWriter<T>::<init>()"
       new: "method void org.apache.iceberg.data.parquet.BaseParquetWriter<T>::<init>()"
       justification: "Changing deprecated code"
+  "1.9.0":
```

Comment:
it's probably best to redo the entire file after https://github.com/apache/iceberg/pull/12930 gets merged